$OpenBSD: patch-cmake_FindReadline_cmake,v 1.2 2014/07/09 20:03:23 zhuk Exp $
Zap the odd hint path suffix, making it find libreadline from base.
Review: https://git.reviewboard.kde.org/r/118957/
Committed: af69daaa5dc810e6bdfe7da0f7631a6b47cc01ee
--- cmake/FindReadline.cmake.orig	Sat Feb 15 21:03:35 2014
+++ cmake/FindReadline.cmake	Thu Jun 26 16:44:31 2014
@@ -2,9 +2,7 @@
 if(READLINE_INCLUDE_DIR AND READLINE_LIBRARY)
 	set(READLINE_FOUND TRUE)
 else(READLINE_INCLUDE_DIR AND READLINE_LIBRARY)
-	FIND_PATH(READLINE_INCLUDE_DIR readline/readline.h
-		/usr/include/readline
-	)
+	FIND_PATH(READLINE_INCLUDE_DIR readline/readline.h)
 	
 	FIND_LIBRARY(READLINE_LIBRARY NAMES readline)
         include(FindPackageHandleStandardArgs)
